scripts/run_lighthouse_tests_test.py

```diff
@@ -622,3 +645,38 @@ def mock_popen(*unused_args: str, **unused_kwargs: str) -> MockTask:  # pylint:
                                             '--mode', 'performance',
                                             '--skip_build',
                                             '--record_screen'])
+
+    def test_run_lighthouse_tests_dockerized(self) -> None:
```

Comment:
This doesn't really explain what's being tested.

Reply:
Done.